### ISSUE TITLE
Bad Performance using PANet pre-trained models

### ISSUE THREAD

**Issue by chunchet-ng** [open-mmlab/mmocr#465]
Hi there,

I downloaded the PANet checkpoint for CTW and used them to carry out inference on CTW test set. The results that I got are significantly lower than what's reported on [this page](https://github.com/open-mmlab/mmocr/blob/main/configs/textdet/panet/README.md). I have been using all default configurations without any changes. 

The results that I got are:
CTW1500
thr 0.3, recall：0.638, precision: 0.636, hmean:0.637
thr 0.4, recall：0.638, precision: 0.636, hmean:0.637
thr 0.5, recall：0.638, precision: 0.636, hmean:0.637
thr 0.6, recall：0.638, precision: 0.636, hmean:0.637
thr 0.7, recall：0.638, precision: 0.636, hmean:0.637
thr 0.8, recall：0.638, precision: 0.636, hmean:0.637
thr 0.9, recall：0.618, precision: 0.671, hmean:0.643
{'hmean-iou:recall': 0.6182113821138211, 'hmean-iou:precision': 0.6705467372134039, 'hmean-iou:hmean': 0.64331641285956}

I suspect that there is something wrong with the released checkpoints, it seems like it is not trained properly.
Based on this [CTW training log file](https://download.openmmlab.com/mmocr/textdet/panet/panet_r18_fpem_ffm_sbn_600e_ctw1500_20210219-3b3a9aa3.log.json), the validation set's H-mean has already reached 0.66035 at 10th epoch, which is higher than what I get through the released checkpoint.
Could you please help me to verify this?

**Comment by ming-eng** [comment]
panet_r18_fpem_ffm_600e_icdar2015 
thr 0.3, recall：0.638, precision: 0.636, hmean:0.637
thr 0.4, recall：0.638, precision: 0.636, hmean:0.637
thr 0.5, recall：0.638, precision: 0.636, hmean:0.637
thr 0.6, recall：0.638, precision: 0.636, hmean:0.637
thr 0.7, recall：0.638, precision: 0.636, hmean:0.637
thr 0.8, recall：0.638, precision: 0.636, hmean:0.637
thr 0.9, recall：0.618, precision: 0.671, hmean:0.643
训练效果太差了
请问你遇到了吗
